source/Outfit.cpp

```diff
@@ -33,6 +33,15 @@ namespace {
 	// disallowed or undesirable behaviors (such as dividing by zero).
 	const auto MINIMUM_OVERRIDES = map<string, double>{
 		// Attributes which are present and map to zero may have any value.
+		{"cooling energy", 0.},
+		{"cloaking energy", 0.},
+		{"cloaking fuel", 0.},
+		{"cloaking heat", 0.},
+		{"cloaking hull", 0.},
+		{"cloaking hull delay", 0.},
+		{"cloaking shield", 0.},
+		{"cloaking shield delay", 0.},
+		{"cloaking action", 0.},
```

Comment:
The comment says attributes in this map which map to 0 may have any value.
What does it mean for `cloaking action` to be negative? What about hull and shield delays?

Reply:
Negative cloaking action means it is free to shoot whilst cloaked (not that it actually gives any cloak to shoot)

Well you may say it isn't truly a cloak anymore in a case of hull and shields being negative.
But I don't think it make sense to reduce the possibilities in plugins and such. If someone puts a negative value there it isn't an error we should not be the ones saying what they should do, what should be forbidden is only what does not work (like a negative cloaking visibility)